docs/web_auth_example/login.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Web Authentication Test Page</title>
+  <meta name="description" content="A test page for web login">
+  <meta name="author" content="kkzeng">
```

Comment:
Hmm not sure whether putting this in the code is a good idea, maybe just put a comment header in the beginning of the file to write license, contributors, etc. ?